melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/BucinfoDto.java

```diff
@@ -29,6 +30,7 @@
     public static BucinfoDto av(BUC buc, List<String> statuser, String rinaUrlPrefix) {
         return BucinfoDto.builder()
                 .id(buc.getId())
+                .bucStatus(buc.getStatus())
```

Comment:
Kanskje det hadde passet like godt med en boolean `erÅpen`..? 🤷 

Har forsøkt å la melosys-eessi oversette det som har med statuser/engelske terms til norsk, for å slippe å foholde oss direkte til RINA-sitt domenespråk. (SedStatus er et eksempel på dette)

Reply:
Funker i alle fall for use-caset jeg har nå. Så kan sikkert det 👍

Reply:
Hehe satser på det forblir slik. Jeg har i hvert fall vanskelig for å se at vi trenger å skille på forskjellige BUC-statuser, i hvert fall ikke i melosys-api. (Er usikker på om det i det hele tatt finnes flere statuser enn open/closed også. Nærmest umulig å finne ut 😛 )